packages/frontend/src/components/attachment/messageAttachment.tsx

```diff
@@ -54,6 +55,22 @@ export default function Attachment({
       openAttachmentInShell(message)
     }
   }
+  /**
+   * we need a fix height for images to prevent reflow when the image loads
+   * till we have a better solution we allow a maximum height of 200px for
+   * all images in landscape mode and 300px for portrait mode
```

Comment:
It sounds like this comment belongs to the CSS part of this MR.
And I'm not sure if the context of the comment is understandable, i.e. a better solution to what problem.

But I guess people can find the commit that introduced this later on.